recipes/hyperscan/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conans import ConanFile, CMake, tools
+
+import os
+
+class HyperscanConan(ConanFile):
+    name = "hyperscan"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.hyperscan.io"
+    description = "High-performance regular expression matching library"
+    topics = ("regex", "regular expressions")
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "optimise": [True, False, "auto"],
```

Comment:
Why auto? (Honestly I would remove this option, turn off underlying value in CMake)